test/Passes/DefaultPipeline/tpp-mapping.mlir

```diff
@@ -216,19 +212,16 @@ func.func @propagate_pack_unpack(%arg0: tensor<128x512xf32>, %arg1: tensor<512x2
 // CHECK: scf.for
 // CHECK:   scf.for
 // CHECK:     tensor.extract_slice{{[^:]+}}: tensor<128x512xf32> to tensor<32x32xf32>
-// CHECK:     linalg.transpose
 // CHECK:     tensor.insert_slice{{[^:]+}}: tensor<32x32xf32> into tensor<4x16x32x32xf32>
 // Generalized pack of the second input
 // CHECK: scf.for
 // CHECK:   scf.for
 // CHECK:     tensor.extract_slice{{[^:]+}}: tensor<512x256xf32> to tensor<32x32xf32>
-// CHECK:     linalg.transpose
```

Comment:
The second input has `outer_dims_perm`, so perhaps there's some additional logic needed here?